attributes/default.rb

```diff
@@ -66,7 +66,18 @@
       '/media/downloadable/',
       '/pkginfo/',
       '/report/config.xml',
-      '/var/'
+      '/var/',
+      '/.gitignore',
+      '/.htaccess',
+      '/.htaccess.sample',
+      '/composer.json',
+      '/cron.php',
+      '/cron.sh',
+      '/index.php.sample',
+      '/install.php',
+      '/mage',
```

Comment:
It is from mage enterprise repository, see https://github.com/inviqa/magento-enterprise/blob/master/mage